packages/web/src/scss/components/Button/_Button.scss

```diff
@@ -81,6 +83,7 @@
 
 .Button--loading {
     position: relative;
+    overflow: hidden; // 1.
```

Comment:
hopefully, it won't blow up somewhere

Reply:
The only use case I can think of is something [like this](https://react-ui.io/components/button#icon-buttons-with-a-badge). But we don't have that in Spirit.

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/8d97dee5-ae13-4566-bca5-d32a3614925b)